.gitignore

```diff
@@ -17,10 +17,6 @@ Thumbs.db
 *_rsa
 *_rsa.pub
 
-# VS Code #
-.vscode
-*.code-workspace
```

Comment:
Why were we ignoring this ones in first place? those are pretty important files that need to be shared

Reply:
Yeah I have no idea, lol. the files inside .vscode were already committed so it makes no sense to have this.